change/@fluentui-react-4910fb47-83b1-40c5-b50c-817986591a43.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32360&commitId=1427c318703230ac62e577625a1142d50fffd325&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|4| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|
|Drawer.Full Overlay RTL.chromium.png|1140| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>